radar-upload-backend/src/test/java/org/radarbase/upload/doa/RecordRepositoryImplTest.kt

```diff
@@ -32,8 +32,7 @@ internal class RecordRepositoryImplTest {
     @BeforeEach
     fun setUp() {
         doaFactory = DoaEntityManagerFactory(
-                Config(jdbcUrl = "jdbc:h2:file:${tempDir.resolve("db.h2")};DB_CLOSE_DELAY=-1;MVCC=true"),
-                mapOf(Pair("hibernate.show_sql", "true"))
```

Comment:
FYI: `show_sql` is useful to investigate whether not too many duplicate fetch requests are made.